### PR TITLE
[docs] Clarify what `AdapterDayjs` is in the Getting Started page

### PR DESCRIPTION
**Pull request by flaviendelangle**

### CODE REVIEW

File: docs/data/date-pickers/getting-started/getting-started.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ yarn add @mui/material @mui/styled-engine-sc styled-components
 ## Code setup
 
 Before trying to render any component, you have to make sure that there is a `LocalizationProvider` upper in the React tree.
-This component receives your chosen date library's adapter and makes it accessible to all the Date and Time Pickers component using React context.
+This component receives your chosen date library's adapter (the doc uses `AdapterDayjs` which is based on [dayjs](https://day.js.org/)) and makes it accessible to all the Date and Time Pickers component using React context.
```
Comment (LukasTy) [inline docs/data/date-pickers/getting-started/getting-started.md:58]: If we are changing it, maybe also add a back-link to `date-library` section?
```suggestion
This component receives your chosen [date library's adapter](https://mui.com/x/react-date-pickers/#date-library) (the doc uses `AdapterDayjs` which is based on [dayjs](https://day.js.org/)) and makes it accessible to all the Date and Time Pickers component using React context.
```

File: docs/data/date-pickers/getting-started/getting-started.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ yarn add @mui/material @mui/styled-engine-sc styled-components
 ## Code setup
```
Comment (joserodolfofreitas) [inline docs/data/date-pickers/getting-started/getting-started.md:55]: Maybe we can change the title to help with searches and give a more meaningful title with what needs to be set up.

```suggestion
## Code setup - The role of adapters
```

Comment (flaviendelangle) [reply docs/data/date-pickers/getting-started/getting-started.md:55]: How about 

```suggestion
## Setup your date library adapter
```

That way we have both the idea of "date library" and of "adapter" to optimize the search results.

Comment (LukasTy) [reply docs/data/date-pickers/getting-started/getting-started.md:55]: I do like this suggestion! 👍

**Review by LukasTy — APPROVED** [review]

**Pull request merged**